### PR TITLE
Issue fixes for RSS feed improvements

### PR DESCRIPTION
**Pull request by n0toose**
Follow-up for #28368

- Just replace button with an a-element with the button class
- Remove useless link-action class from template/org/home.tmpl

### CODE REVIEW

**Review by wxiaoguang — APPROVED** [review]
Just a nit, I would argue whether `role="button"` really makes sense, because it is really a "link": it has `href`, it navigates ....

File: templates/org/home.tmpl [hunk 1]
```diff
@@ -21,9 +21,9 @@
 		</div>
 		<div class="right menu">
 			{{if .EnableFeed}}
-			<button class="link-action ui basic label button gt-mr-0" data-tooltip-content="{{ctx.Locale.Tr "rss_feed"}}" data-url="{{$.Org.HomeLink}}.rss">
+			<a class="ui basic label button gt-mr-0" role="button" href="{{$.Org.HomeLink}}.rss" data-tooltip-content="{{ctx.Locale.Tr "rss_feed"}}">
```
Comment (wxiaoguang) [inline templates/org/home.tmpl]: Another nit, I don't know what the `label` class it used for (does `ui basic button` work?)

**Comment by n0toose** [comment]
> Just a nit, I would argue whether role="button" really makes sense, because it is really a "link": it has href

You're right, I thought that I would add that because it visually looks like a button, but this is essentially the same link as before in a trenchcoat of a button.

File: templates/org/home.tmpl (hunk 1 above)
Comment (n0toose) [reply templates/org/home.tmpl]: I have a reasonable (and possibly cursed) explanation for this, I think it made the icon scale the way I wanted it to exactly. I will test this again and confirm / deny it.

Comment (n0toose) [reply templates/org/home.tmpl]: OK, so it looks like this without the `label`.

image

This "method" of fixing this was basically borrowed from the repository header file, where I also applied a similar solution for the same reason.

Comment (wxiaoguang) [reply templates/org/home.tmpl]: No problem. Maybe it's caused by other strange CSS styles ... then let's keep it.

Comment (n0toose) [reply templates/org/home.tmpl]: By the way, thanks for the patience.

**Review by jolheiser — APPROVED** [review]
Thanks for the follow-up!

**Pull request merged**